src/DotNetProjectFile.Analyzers/DotNetProjectFile.Analyzers.csproj

```diff
@@ -44,6 +44,8 @@
 
   <ItemGroup Label="Roslyn dependencies">
     <PackageReference Include="Microsoft.CodeAnalysis.Common" Version="4.10.0" ExcludeAssets="runtime" />
+    <PackageReference Include="NuGet.Configuration" Version="6.13.1" />
+    <PackageReference Include="NuGet.Packaging" Version="6.13.1" />
```

Comment:
I don't think we really need this dependency to be honest, it is quite straightforward to find the .nuspec file and in there we honestly only care about 1 or 2 fields `package` -> `metadata` -> `license` and/or `package` -> `metadata` -> `licenseUrl`